api/app/Models/Localization.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Models;
+
+use Illuminate\Support\Facades\Cache;
+use Spira\Model\Model\BaseModel;
+use Spira\Model\Model\CompoundKeyTrait;
+
+class Localization extends BaseModel
+{
+    use CompoundKeyTrait;
+
+    const cacheKeyBuilder = 'l10n:%s:%s';
+
+    // This is a duplicate of the supportedRegions object found in regionService.ts
+    public static $supportedRegions = [
+        [
+            'code' => 'au',
+            'name' => 'Australia',
+            'icon' => '&#x1F1E6;&#x1F1FA;',
+        ],
+        [
+            'code' => 'uk',
+            'name' => 'United Kingdom',
+            'icon' => '&#x1F1EC;&#x1F1E7;',
+        ],
+        [
+            'code' => 'us',
+            'name' => 'United States',
+            'icon' => '&#x1F1FA;&#x1F1F8;',
+        ],
+    ];
+
+    /**
```

Comment:
There needs to be validation on this model so that when a put request is sent to create a new localisation the localisations array is validated to be an array (object) and the localisation key is one of the accepted values.